pyproject.toml

```
@@ -40,8 +41,7 @@ dependencies = [
     "datasets",
     "qwen_vl_utils",
     "vllm==0.10.2",
-    "optimum-rbln>=0.9.3",
-    "numpy<2"
+    "optimum-rbln>=0.9.4a4",
```

```suggestion
    "optimum-rbln>=0.9.4a4",
    "torch==2.8.0"
```
@rebel-sunwook  How about removing transitive dependency?